### DISCUSSION TITLE
Delivery Receipts and Read Receipts

### DISCUSSION THREAD

**Discussion by SufiyanM** [postalserver/postal#2614] [Feature suggestions]
I have consumed Postal Open APIs and webhooks, now I want to get delivery receipts and read receipts for the emails sent. Kindly assist how it would be done. Thanks!

**Comment by willpower232** [comment]
You can combine Click and Open Tracking with Webhooks to receive all the different kinds of event in your webhook

https://docs.postalserver.io/features/click-and-open-tracking

Technically it isn't a read or delivery receipt but its pretty close.
